pkg/metadata/annotations.go

```diff
@@ -144,6 +144,16 @@ const (
 	// reconciler-manager reads it. If set to true, the reconciler-manager will
 	// create the reconciler with the Namespace controller in the reconciler container.
 	DynamicNSSelectorEnabledAnnotationKey = configsync.ConfigSyncPrefix + "dynamic-ns-selector-enabled"
+
+	// ImageURLAnnotationKey is the annotation key applied to R*Sync objects
+	// when pre-sync is either disabled or successfully completed.
+	// This annotation stores the source image URL.
+	ImageURLAnnotationKey = configsync.ConfigSyncPrefix + "image-url"
+
+	// SourceCommitAnnotationKey is the annotation key applied to R*Sync objects
+	// when pre-sync is either disabled or successfully completed.
```

Comment:
The comment is still confusing. It interprets like the annotation is applied only when pre-sync is disabled or successfully completed.

What about the following?
```
// SourceCommitAnnotationKey stores the Git source commit fetched by git-sync, the OCI image digest fetched by oci-sync, or the Helm chart version fetched by helm-sync.
// It can be used for OCI signature verification.
```

```

Reply:
Done